src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -1149,5 +1149,53 @@ public void DataClassAndStruct()
                 Diagnostic(ErrorCode.ERR_LocalDuplicate, "Y").WithArguments("Y").WithLocation(5, 27)
             );
         }
+
+        [Fact]
+        public void RecordInheritance()
```

Comment:
>RecordInheritance [](start = 20, length = 17)

Can you also include non-class types deriving from record? (interface, struct, enum)